server/api/[version]/distribution.ts

```diff
@@ -21,12 +21,12 @@ export default defineCachedEventHandler(async () => {
     )
     .get()
 
-  const staked = result?.totalStaked ?? 0
+  const staked = (result?.totalStaked ?? 0) * 100
```

Comment:
I need to multiply this by 100 to have the real value:

```
{
  "staked": 58506711040250200,
  "circulating": 134161808883037360,
  "ratio": 0.43609065446677536
}
```

On the other hand, https://nim.sh/stats/supply.txt returns around 49%